lib/fare.rb

```diff
@@ -0,0 +1,19 @@
+class Fare
+
+  PENALTY_FARE = 6
+
+  def self.calculate(journey = {})
```

Comment:
Personally I wouldn't have a class method here, but instead use this to build an object

```
self.new(journey).calculate
```

the reason being that the fact you are passing journey to the private methods is a code smell and an indicator that that should be part of the state of the fare object. 

By doing that you don't need to pass journey as an argument to your private methods and your code is cleaner